tests/include.py

```diff
@@ -36,39 +39,55 @@ def add_required_env_arguments(benchmark_specs, config, env, master_nodes_list):
         config['redis_process_port'] = master_nodes_list[0]['port']
 
 
-def debugPrintMemtierOnError(config, env):
```

Comment:
@filipecosta90 I think you reverted the prev. commit by mistake.

Reply:
@ushachar you're absolutely right. fixed now :)